interface/client/styles/popupWindows.import.less

```diff
@@ -9,6 +9,28 @@
         color: @colorLinkFocus;
     }
 
+    &.update-available {
+        .text {
+            text-align: left;
+        }
+
+        h1 {
+            font-size: 200%;
+            margin-bottom: 20px;
+
+            &.no-upate {
+                font-size: 180%;
+            }
+        }
+
+        #image {
```

Comment:
id's are a no go :) especially if you nest it them in a class. i would simply target the image, or use a class